packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -49,6 +49,7 @@
         align-items: center;
         background: var(--general-main-2, #ffffff);
         z-index: 1;
+        border-radius: 0.8rem 0.8rem 0 0;
```

Comment:
@aizad-deriv I think we just have to remove background from `wallets-modal-step-wrapper__header`